ilastik/plugins_default/tracking_mamut_export.py

```diff
@@ -70,38 +70,65 @@ def export(self, filename, hypothesesGraph, objectFeaturesSlot, labelImageSlot,
         graph = hypothesesGraph._graph
 
         features = objectFeaturesSlot([]).wait() # this is a dict of structure: {frame: {category: {featureNames}}}
-        firstPass = True
 
+        # first loop over all nodes to find present features
+        presentTrackIds = set([])
         for node in graph.nodes_iter():
             frame, label = node
+            # print(f"Node {node} has value={graph.node[node]['value']} and trackId={graph.node[node]['trackId']}")
             if graph.node[node]['value'] > 0:
                 t = graph.node[node]['traxel']
 
                 radius = 2*features[frame]['Standard Object Features']['RegionRadii'][label, 0]
```

Comment:
`radius` is not used in this scope. Also I'm kind of wondering, why radius is multiplied by 2...